src/assets/styles/style.less

```diff
@@ -15,7 +15,7 @@
 @import "../../sidebars/favourites/minichart/minichart.less";
 @import "../../sidebars/star/star.less";
 @import "../../showcase/showcase-changes.less";
-@import "custom-scroll.less";
+@import "../../sidebars/custom-scroll.less";
```

Comment:
The reason for the move was that we're only using it in the sidebar, and every time I went to find it I had to search around for not knowing where it was!

Happy to submit as part of another PR though, or leave where it is.